### PR TITLE
Fixed SystemNotSupportedException caused by thirdparty mod DLLs on .NET > 4.0

### LINKED ISSUE #8156: 3rd-party mod dlls need custom config on Windows systems with older .NET
On Windows machines which don't have .NET versions newer than 4.0 installed, a custom OpenRA.Game.exe.config with this content is needed to avoid a ```SystemNotSupportedException``` with custom, directly downloaded 3rd-party mod DLLs:

http://www.codeitive.com/0HmeqUeeVq/c-configure-loadfromremotesources-through-code-not-appconfig.html

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/14670314-3rd-party-mod-dlls-need-custom-config-on-windows-systems-with-older-net?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Mailaender**
Fixes https://github.com/OpenRA/OpenRA/issues/8156 (untested).

Harvested the file from http://www.ppmforums.com/viewtopic.php?t=39967.

### CODE REVIEW

File: OpenRA.Game.exe.config [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (pchote) [inline OpenRA.Game.exe.config]: Could we put this in packaging/windows?

Comment (Mailaender) [inline OpenRA.Game.exe.config]: I assume it will then not work when you compile from bleed and throw a thirdparty mod DLL in there.

Comment (pchote) [inline OpenRA.Game.exe.config]: I was under the impression that this was only a problem for shipped installs, but I may be wrong.

Comment (pchote) [inline OpenRA.Game.exe.config]: If it is a problem for self-compiles, then we could have the windows dependencies script copy it to the root.

Comment (Mailaender) [inline OpenRA.Game.exe.config]: Does this break anything on Mono/OS X for you?

**Comment by Mailaender** [comment]
Fix has been confirmed at https://www.facebook.com/openra/posts/891169000939318?comment_id=891198397603045&reply_comment_id=891212777601607&total_comments=2&comment_tracking=%7B%22tn%22%3A%22R%22%7D

File: OpenRA.Game.exe.config (hunk 1 above)
Comment (Mailaender) [inline OpenRA.Game.exe.config]: I may also need this file for https://github.com/OpenRA/OpenRA/pull/8196.

**Comment by reaperrr** [comment]
Pinging @abcdefg30 for confirmation.

**Comment by abcdefg30** [comment]
Yes, it works indeed. ;) :+1: / :white_check_mark:

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/2b077437c2027a2aee0c74c7c8c996a6fecc1d5a...3ecfa0580cc3bd1d6607bb8e3555c2345545b302)

**Pull request merged**